fameal/ape-tax
# Feature/new tag

## What it does ✨
Add a new tag `new` to highlight the new vaults.  Color is set to #10B981, but there is nothing definitive.  
Update the script to fix an issue with defaultValue + update the new/active vault flow : 
- A vault is now `new` by default
- When adding another vault, the previous `new` become `active` only.

![Capture d’écran 2021-08-05 à 12 52 45](https://user-images.githubusercontent.com/9974362/128347576-9110923b-8f86-4b2e-85ea-7bdfb5fba1b9.png)


## How to test ✅
By putting the `new` tag in `VAULT_STATUS`, a new tag should be visible on the vault list.  
Running the script to add a new vault should remove the old `new` and set `new` to the new vault.
